---
layout: archive
title: "Research"
permalink: /research/
author_profile: true
---

{% include base_path %}

I use a combination of **genome-resolved metagenomics**, **phylogenetics**, and **experimental evolution** to understand how microbial communities adapt to their environments and drive global biogeochemical cycles.

## Climate Solutions & Soil Microbiomes
*Current work at the Innovative Genomics Institute, UC Berkeley (<a href="https://www.banfieldlab.com/" target="_blank">Jill Banfield Lab</a>)*

My postdoctoral research applies genome-resolved metagenomics & metatranscriptomics to evaluate the potential of soil microbial communities to mitigate climate change.

* **Carbon Sequestration in Agricultural Soils**
    * Evaluating the impact of soil amendments on microbial community structure and function in California rangelands.
    * Integrating metagenomics and metatranscriptomics with biogeochemical measurements to assess greenhouse gas flux (CO2, CH4, N2O), carbon retention, and soil health.
    * *Collaboration with Dr. Whendee Silver (UC Berkeley).*

---

## Evolutionary Dynamics of Specialized Metabolism
*PhD Research at Scripps Institution of Oceanography (Paul Jensen Lab)*

My doctoral work focused on the evolution of biosynthetic gene clusters (BGCs) in the marine actinomycete *Salinispora* and the development of computational tools to mine genomic data for natural products.

* **Phylogenomics of *Salinispora***
    * Performed whole-genome sequencing of 99 new *Salinispora* strains on a microscale spatial scale to investigate fine-scale taxonomic and biosynthetic diversity.
    * Described the evolutionary radiation of ribosomally synthesized and post-translationally modified peptides (RiPPs) within the *Salinispora* populations.
    * Evolutionary analysis of the *Salinispora* salinipostin biosynthetic gene cluster, uncovering new diversity in bacterial signaling molecule biosynthetic potential.

* **NaPDoS2: Natural Product Domain Seeker**
    * Developed and validated <a href="https://npdomainseeker.sdsc.edu/napdos2/" target="_blank">NaPDoS2</a>, a webtool that uses phylogeny-based classification to detect ketosynthase (KS) and condensation (C) domains in genomic data.
    * Applied NaPDoS2 to measure polyketide diversity by uncovering >60,000 KS domains, revealing novel patterns of biosynthetic potential across the entire tree of life.

* **Deep-Sea Exploration on E/V Nautilus (2020)**
    * **Shipboard Scientist** on the *NA124* expedition to the Southern California Borderland.
    * Assisted ROV-mediated sample collections to assess the biopharmaceutical potential of benthic deep-sea ecosystems.
    * Conducted shipboard sample processing, including genomic and metabolomic preservation.
    * <a href="https://www.youtube.com/watch?v=_y3MAH65Cfg&list=PL41b0O3MiKnYOFB_9lRdvgSUA95pG2WAV" target="_blank">Full Dive videos on YouTube</a>

* **Advanced Bacterial Genetics (Cold Spring Harbor Laboratory) (2019)**
    * **Course Participant** (June 2019): Intensive training in transposon mutagenesis (Tn-Seq), CRISPR/Cas genome editing, and recombineering in *E. coli*, *Salmonella*, and *Vibrio*.

---

## Experimental Evolution & Stress Response
*Early Career Research at Kenyon College, JCU, & The Whitney Laboratory for Marine Bioscience*

My foundational research used experimental evolution and physiological assays to study how organisms adapt to specific chemical, thermal, and environmental stressors.

* **Adaptation to Acid Stress in *E. coli* (Kenyon College) (2012-2016)**
    * Investigated the fitness and genomics of *Escherichia coli* K-12 strains evolved under aromatic acid stress (benzoic acid).
    * Demonstrated how long-term exposure to stress leads to specific genetic adaptations, culminating in a thesis awarded *Highest Honors*.

* **Amphibian Pathogen Ecology (James Cook University) (2015)**
    * **Independent Study Project:** "The effect of frog body temperature on the growth of the amphibian chytrid fungus *Batrachochytrium dendrobatidis*."
    * Investigated how frog-specific elevation-dependent temperature regimes impact the long-term growth of the pathogenic chytrid fungus.
    * *Assistant to senior graduate student who led the project: Sasha E. Greenspan, supervised by Dr. Ross A. Alford*

* **Developmental Signaling in *Nematostella* (The Whitney Laboratory for Marine Bioscience) (2015)**
    * **NSF REU Fellow:** Studied the role of BMP signaling in patterning the primary and secondary axis of the starlet sea anemone *Nematostella vectensis*.
    * Used CRISPR/Cas9 embryo microinjections and *in situ* hybridization to propose a new regulation model of body axis development.
    * *Supervised by: Dr. Mark Q. Martindale & senior graduate student David Simmons.*
